deps/chakrashim/src/v8objecttemplate.cc

```diff
@@ -85,20 +63,21 @@ class ObjectTemplateData : public TemplateData {
   }
 
   bool AreInterceptorsRequired() {
-    return namedPropertyDeleter != nullptr ||
-      namedPropertyEnumerator != nullptr ||
-      namedPropertyGetter != nullptr ||
-      namedPropertyQuery != nullptr ||
-      namedPropertySetter != nullptr ||
-      namedPropertyDefiner != nullptr ||
-      namedPropertyDescriptor != nullptr ||
-      indexedPropertyDeleter != nullptr ||
-      indexedPropertyEnumerator != nullptr ||
-      indexedPropertyGetter != nullptr ||
-      indexedPropertyQuery != nullptr ||
-      indexedPropertySetter != nullptr ||
-      indexedPropertyDefiner != nullptr ||
-      indexedPropertyDescriptor != nullptr;
+    return setterGetterInterceptor != nullptr &&
```

Comment:
If we only assign the `setterGetterInterceptor` pointer if it is used, perhaps this condition can collapse to just this first check?

Reply:
This is already a pretty cheap call both ways. Also `setterGetterInterceptor` is presumably `nullptr` majority of the time.